recipes/lora_finetune_distributed.py

```diff
@@ -304,11 +311,11 @@ def _setup_model(
             sync_module_states=True,
             # Initialize empty modules on all non-zero ranks
             param_init_fn=(
-                lambda module: module.to_empty(
-                    device=torch.device("cuda"), recurse=False
+                lambda module: (
```

Comment:
These lint changes are unfortunate...we should really sync up the linting with what VSCode does.

Reply:
I still don't really understand this, if you've installed pre-commit hooks then lint when you commit locally should be the same as the linter on CI, no?